test/mindee.js

```diff
@@ -27,10 +27,9 @@ describe('Create - Mindee_invoice', () => {
       inputData: {},
     };
 
-    const result = await appTester(
```

Comment:
I don't know why you had to change the promise from the new syntax (async await) to the old syntax (then catch). Is it necessary ?

Reply:
There is something with Moca & Zapier Testing Class, it was giving me a deprecated error (not a warning)

`[DEP0097] DeprecationWarning: Using a domain property in MakeCallback is deprecated. Use the async_context variant of MakeCallback or the AsyncResource class instead. (Triggered by calling processImmediate on process.)`

Reply:
A change since Node.js 14.something

Reply:
Okey, I m not sure why but you can keep the old syntax if it solve the deprecation issue

Reply:
Me neither!